### PR TITLE
Issue-124 Deprecate the WavefrontProxyClient in favor of a single cli…

### PR DESCRIPTION
**Pull request by MikeMcMahon**
…ent for sending or a Delegating Facade for multi client

This change depends on our LATEST proxy version that supports all formats on the same port.

### CODE REVIEW

File: src/main/java/com/wavefront/sdk/Main.java [hunk 1]
```diff
@@ -103,13 +104,18 @@ private static void sendTracingSpan(WavefrontSender wavefrontSender)
   }
 
   public static void main(String[] args) throws InterruptedException, IOException {
+
+
```
Comment (basilisk487) [inline src/main/java/com/wavefront/sdk/Main.java:108]: nit: blank lines

File: src/main/java/com/wavefront/sdk/common/clients/service/ReportingService.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    this.token = token;
+  }
+
+  @Override
+  public int send(String format, InputStream stream) throws IOException {
+    HttpURLConnection urlConn = null;
+    int statusCode = 400;
+    try {
+      String originalPath = uri.getPath() != null ? uri.getPath() : "";
+      URL url = new URL(uri.getScheme(), uri.getHost(), uri.getPort(), originalPath + "?/f=" + format);
```
Comment (basilisk487) [inline src/main/java/com/wavefront/sdk/common/clients/service/ReportingService.java:37]: shouldn't it be "/?f=" ?

Comment (MikeMcMahon) [reply src/main/java/com/wavefront/sdk/common/clients/service/ReportingService.java:37]: GOOD CATCH

**Comment by sushantdewan123** [comment]
lgtm. but this change is not ready to be merged to master until end of this week.
I will merge this to master around Friday.

**Pull request merged**